bes_theme/assets/data/version_details/151-hexo-Versiondetails.json

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "version": "6.2.0",
+        "release_date": "11-May-2022",
+        "criticality_score": "Not Available",
```

Comment:
criticality_score and scorecard details are not available?

Reply:
> criticality_score and scorecard details are not available?

I don't know how to do it, today I will to.